biome.json

```diff
@@ -90,6 +90,16 @@
           }
         }
       }
+    },
+    {
+      "include": ["packages/ntds/**"],
+      "linter": {
+        "rules": {
+          "a11y": {
+            "noSvgWithoutTitle": "off"
```

Comment:
If you're turning this off, why do all of the generated SVG components have title handling?